exportAwsCredentials.js

```diff
@@ -1,20 +1,12 @@
 const fs = require("fs");
 const ini = require("ini");
-const credentialsFilePath = "/[YOUR HOME HERE]/.aws/credentials";
+const homedir = require('os').homedir();
+const credentialsFilePath = `${homedir}/.aws/credentials`;
 const awsProfile = process.argv[2] || "edrans";
+const credentialsIni = ini.parse(fs.readFileSync(credentialsFilePath, "utf-8"))[awsProfile];
 
-console.log(awsProfile);
-const credentialsIni = ini.parse(fs.readFileSync(credentialsFilePath, "utf-8"));
-
-console.log(
-  "export AWS_ACCESS_KEY_ID=%s",
-  credentialsIni[awsProfile].aws_access_key_id
-);
-console.log(
-  "export AWS_SECRET_ACCESS_KEY=%s",
-  credentialsIni[awsProfile].aws_secret_access_key
-);
 console.log(
-  'export AWS_SESSION_TOKEN="%s"',
-  credentialsIni[awsProfile].aws_session_token
+  `export AWS_ACCESS_KEY_ID=${credentialsIni.aws_access_key_id}`,
+  `AWS_SECRET_ACCESS_KEY=${credentialsIni.aws_secret_access_key}`,
+  credentialsIni.aws_session_token ? `AWS_SESSION_TOKEN=${credentialsIni.aws_session_token}`:''
```

Comment:
nice! You need to escape this string in " when printed because it may contain characters that bash might interpret.

Reply:
Are you referencing in your question the structuring part? When I use for instance ${credentialsIni.aws_access_key_id}

Do you mean in a near future because you might to add a new feature?